tests/test_utils.py

```diff
@@ -1506,3 +1506,33 @@ def test_exception_raised_with_details(self):
             " the LULC raster but not the table are: [3].")
         self.assertTrue(
             expected_message in str(context.exception), str(context.exception))
+
+class CheckArrayNodataTests(unittest.TestCase):
+    """Tests for natcap.invest.utils.check_array_for_nodata."""
```

Comment:
If we rename the func we may want to rename this class as well.

Reply:
Yep: https://github.com/natcap/invest/pull/770/commits/9876abdde38bb2075ee70cef3c76d5b61f808d4d